common/src/main/java/com/wynntils/screens/partymanagement/PartyManagementScreen.java

```diff
@@ -0,0 +1,426 @@
+     */
+    private void reloadMembersWidgets() {
+        partyMembersWidgets.clear();
+        List<String> partyMembers = new ArrayList<>(Models.Party.getPartyMembers());
+        for (int i = 0; i < partyMembers.size(); i++) {
+            String playerName = partyMembers.get(i);
+            if (playerName == null) continue;
+
+            partyMembersWidgets.add(new PartyPlayer(
+                    this.width / 2 - X_START + 4,
+                    this.height / 2 - 125 + i * 20 - 10,
+                    200,
+                    20,
+                    playerName,
+                    offlineMembers.contains(playerName)));
+        }
+    }
+
+    @SubscribeEvent
+    public void onPartyUpdate(RelationsUpdateEvent.PartyList e) {
+        reloadMembersWidgets();
+        reloadSuggestedPlayersWidgets(); // Reload because we don't want to suggest party members
+    }
+
+    private void inviteFromField() {
+        // Remove all except commas, semicolons, whitespaces, and characters possible in name
+        String fieldText =
+                INVITE_REPLACER.matcher(inviteInput.getTextBoxInput()).replaceAll("");
+        fieldText = COMMA_REPLACER.matcher(fieldText).replaceAll(","); // semicolons and spaces to comma
+        if (fieldText.isBlank()) return;
+
+        if (!Models.Party.isPartying()) {
+            createParty();
+        }
+
+        Set<String> toInvite = new HashSet<>(List.of(fieldText.split(",")));
+        toInvite.removeAll(Models.Party.getPartyMembers());
+        toInvite.forEach(playerName -> McUtils.sendCommand("party invite " + playerName));
+
+        inviteInput.setTextBoxInput("");
+    }
+
+    private void refreshParty() {
+        Models.Party.requestPartyListUpdate();
+        reloadSuggestedPlayersWidgets();
+        reloadMembersWidgets();
+        refreshOffline();
+    }
+
+    private void kickOffline() {
+        refreshParty();
+        offlineMembers.forEach(playerName -> McUtils.sendCommand("party kick " + playerName));
+    }
+
+    private void refreshOffline() {
+        offlineMembers.clear();
+        offlineMembers.addAll(Models.Party.getPartyMembers());
+        offlineMembers.removeAll(McUtils.mc().level.getScoreboard().getTeamNames());
+    }
+
+    private void createParty() {
+        McUtils.sendCommand("party create");
+    }
+
+    private void leaveParty() {
+        McUtils.sendCommand("party leave");
+    }
+
+    @Override
+    public boolean mouseClicked(double mouseX, double mouseY, int button) {
+        super.mouseClicked(mouseX, mouseY, button);
+
+        // For some reason the (widget.isMouseOver) check does not work for partyMembersWidgets
+        partyMembersWidgets.forEach(widget -> widget.mouseClicked(mouseX, mouseY, button));
+        suggestedPlayersWidgets.forEach(widget -> {
+            if (widget.isMouseOver(mouseX, mouseY)) widget.mouseClicked(mouseX, mouseY, button);
+        });
+
+        return true;
```

Comment:
See comments above, technically this means we have failed to click anything, so return false. Also, why do we not call super here?